frontend/discourse/admin/components/site-settings/date.gjs

```diff
@@ -0,0 +1,10 @@
+import { Input } from "@ember/component";
```

Comment:
Would be good to add a component site setting for this too, there are already a bunch of site setting input ones in https://github.com/discourse/discourse/blob/83fd6153e9f2752094d649c8ee4a61297da86cf6/frontend/discourse/tests/integration/components/site-setting-test.gjs